corehq/apps/userreports/management/commands/clean_report_config.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import OrderedDict
+from copy import deepcopy
+from io import open
+import simplejson as json
+import six
+import textwrap
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.userreports.models import ReportConfiguration, StaticReportConfiguration
+from corehq.apps.userreports.reports.filters import specs
+
+COLUMN_PARAMS_ORDER = [
+    'display',
```

Comment:
Thoughts on making `comment` first? I would hope that if a comment is given, it would be the most useful thing in the definition (though I've tried making column_id == the app's question id so that comments aren't as necessary)

Reply:
I'm in favor of that.  Right now I believe any comments will actually be stripped, since they're not a defined field on report filters or columns (I was also thinking mostly of using this on reports that haven't seen much/any manual editing yet).  I don't think there's any harm in having them there though, and agree that they should be first.

There's probably a lot more that could be done for ordering, like a specific ordering defined for each type of filter/column, but that felt like a bit of a rabbit hole, so I kept it simple to start.  That said, if you find yourself wanting to use this, feel free to make whatever changes you like.